src/components/Flex/__tests__/__snapshots__/flex-snapshot-tests.jest.js.snap

```diff
@@ -0,0 +1,199 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Flex renders correctly Horizontal display with children 1`] = `
+<div
+  className=""
```

Comment:
for some reason it does not add the class names to the test... what should I do?

Reply:
@hadasfa where did you try the add a test with `className`